knowledge-base/kb-window-howto-use-radwindow-as-mainwindow

```diff
@@ -0,0 +1,81 @@
+---
+title: How to Use RadWindow as MainWindow
+description: This article explains how you can replace the main Window with a RadWindow.
+type: how-to
+page_title: How to Use RadWindow as MainWindow
+slug: kb-window-howto-use-radwindow-as-mainwindow
+position: 0
+tags: radwindow,as,mainwindow
+res_type: kb
+---
+
+ ## Environment
+<table>
+	<tr>
+		<td>Product Version</td>
+		<td>2019.1.220</td>
+	</tr>
+	<tr>
+		<td>Product</td>
+		<td>RadWindow for WPF and Silverlight</td>
+	</tr>
+</table>
+
+ ## Description
+
+ This article explains how you can replace the main Window with a RadWindow.
```

Comment:
Use the customer context here. I mean the customer wording. If this is not coming from a customer, you can think how someone would search for this and use this as a base for your description. For example "How to replace Window with RadWindow", or "How to show RadWindow as MainWindow", etc.